app/assets/stylesheets/views/_homepage.scss

```diff
@@ -214,7 +214,7 @@
 }
 
 .chevron-card__description {
-  margin: 0;
+  margin: 0 govuk-spacing(-6) 0 0;
```

Comment:
This might seems a strange way of doing it - after all, why not remove the padding-right on `.chevron-card`?

Reasoning is:

- we can't change the right spacing on the heading, as that's a component
- adding padding right to the link (`.chevron-card__link`) affects the shape of the focus state
- adding margin right to the link doesn't do anything, as it's an inline element
- adding an extra wrapper around the heading and link seems overkill

So we're left with this. 

I checked that it works in the browser that are supported, and well as IE10 and IE9.